src/common/config.go

```diff
@@ -248,6 +255,16 @@ func checkConf() error {
 		return fmt.Errorf("Unknown Sandbox type '%s'", Conf.Sandbox)
 	}
 
+	if Conf.Trace.Format != "default" {
+		if Conf.Trace.Format == "text" || Conf.Trace.Format == "json" {
```

Comment:
So the two formats are "default" and "text"?  I haven't looked at this carefully, but I figured JSON would be one of the options.  I don't like "default" as one of the options because it is not obvious what it is.